### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by yayaismaail**
In this milestone I:
set up React environment using CRA
set up <Router> and
Books and Categories components were set up with their necessary contents
Form component was created
The Navbar component was set up to include Links
Readme file was modified
Errors were fixed using linters

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const Form = () => (
+  <>
+    <form>
+      <input type="text" placeholder="Book Title" />
+      <input type="text" placeholder="Author Name" />
+    </form>
```
Comment (Owusu-Desmond) [inline src/components/Form.js]: - [ ] Please make sure that your form component has a button to submit the button wouldn't add the book *yet*, this is just a placeholder for now :pray:

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import Form from './Form';
+
+const Books = ({ title, author }) => (
+  <>
+    <div>
+      <h2 className="book">{title}</h2>
+      <p className="by">by</p>
+      <p className="author">{author}</p>
+      <button type="button" className="remove-btn">Delete</button>
+    </div>
```
Comment (Owusu-Desmond) [inline src/components/Books.js]: - [ ] Please ensure that your `<Books>` component, contains the following components:

  * `<BookList>`
    * Renders a list of books using the `<Book>` component
  * `<Book>`
    * Takes the book data as a prop and renders the title and the author
    * Has a button for removing books
      * the button wouldn't remove the books *yet*, this is just a placeholder for now

**Review by Owusu-Desmond — APPROVED** [review]
Hi @yayaismaail , 

**Please note that this review is changes required  ♻️**

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 2 above)
Comment (yayaismaail) [reply src/components/Books.js]: @Owusu-Desmond, in the project requirements, microverse did not mention interactivity, it is just a set up(initialization).

Comment (yayaismaail) [reply src/components/Books.js]: @Owusu-Desmond, take a look Screenshot_20230929-192113

Comment (VICTOR-CHUKWU) [reply src/components/Books.js]: - [ ] Just as the previous reviewer mentioned, there should be a component called `BookList` which renders another component called `Book` with props, so what you can do here is two you can create a new component, call it BookList or anything you like, create an array of books in it, map through the array and render this particular component, then import the BookList in you App.js, or you turn this into the booklist component and the create a new Book component that will take props and render it here

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @yayaismaail , 
## STATUS: CHANGES REQUIRED :recycle: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yayaismaail** [comment]
@VICTOR-CHUKWU, please I need your assistance because I am new to react and at this point I don't know how to implement that.
Thanks in anticipation.

**Comment by VICTOR-CHUKWU** [comment]
@yayaismaail  what is your zoom name so I can call you

**Comment by yayaismaail** [comment]
@VICTOR-CHUKWU yayaismail

File: src/components/BookList.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const BookList = () => (
+  <form className="newbook-form">
+    <input type="text" id="inputText" placeholder="Book Name" />
+    <input type="text" className="author-input" placeholder="Author Name" name="author" id="author" />
+    <button type="button" className="add-book">Add Book</button>
+  </form>
+);
```
Comment (thecodechaser) [inline src/components/BookList.js]: - [ ] You created the add book form in the `BookList` component, This component should called `AddBook`, and in `BookList` please use the array of books in it that is available in `Books`, map through the array, and render the `Books` component, then import the BookList in you `App`. Please make sure to fix this issue as requested by the previous reviewer.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @yayaismaail👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookList.js (hunk 3 above)
Comment (yayaismaail) [reply src/components/BookList.js]: @thecodechaser , please I need your help
I am my extra review and it is Saturday

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @yayaismaail ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Impelemented requested changes ✔️
- Used react-router for SPA ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**